[Autocomplete] implement max_results option (#478)
symfony/ux: src/Autocomplete/src/Resources/doc/index.rst

            $qb->andWhere('entity.name LIKE :filter OR entity.description LIKE :filter')
                ->setParameter('filter', '%'.$query.'%');
        }
``max_results`` (default: 10)

>     Allow you to control the reults number returned by a query on automatic endpoint.

Reviewer: ```suggestion
    Allow you to control the max number of results returned by the automatic autocomplete endpoint.
```

Author: Done in 40c7632

>     Set it to ``null`` if you don't want to limit results. 

Reviewer: blank line after this line

Author: Done in 40c7632
